inst/@sym/sym.m

```diff
@@ -277,7 +277,6 @@
   end
 
   asm = {};
-  check = true;
```

Comment:
Would adding this comment address your concern?
````
% Note: a previous version of this code deferred most calls to `python_cmd
% until near the end of the function.  Now, it is called in many places.  Its
% possible this has some negative consequences for a future port to classdef.
% If you're reading this in the future and have successfully done that port,
% then please delete this comment ;-)
````

Otherwise, what should it say?  (I don't understand what the issue is here so harder for me to make a precise statement).